test/e2e/tests/carousel/carousel.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { strict as assert } from 'assert';
```

Comment:
These e2e fixes were needed as I found that carousel at any point displays only the first 5 slides, as we dismiss the slides more load.